test/functional/rpc_packages.py

```diff
@@ -369,14 +369,20 @@ def test_submit_child_with_parents(self, num_parents, partial_submit):
     def test_submitpackage(self):
         node = self.nodes[0]
 
+        self.log.info("Submitpackage only allows valid hex inputs")
+        valid_hex = self.wallet.create_self_transfer()["hex"]
+        assert_raises_rpc_error(-22, "TX decode failed:", node.submitpackage, [valid_hex, valid_hex[:-1] + 'X'])
+
         self.log.info("Submitpackage valid packages with 1 child and some number of parents")
         for num_parents in [1, 2, 24]:
             self.test_submit_child_with_parents(num_parents, False)
             self.test_submit_child_with_parents(num_parents, True)
 
+
         self.log.info("Submitpackage only allows packages of 1 child with its parents")
         # Chain of 3 transactions has too many generations
         legacy_pool = node.getrawmempool()
+
```

Comment:
Are these blank lines intentional?

Reply:
I removed them in [1fab1d6](https://github.com/bitcoin/bitcoin/pull/31139/commits/1fab1d601e40edc7296cf3d60a7e917b0e6493b3)